tests/test_confidence.py

```diff
@@ -0,0 +1,89 @@
+from vulture import core
+
+from . import skip_if_python_2
+from . import skip_if_python_3
+
+dc = core.DEFAULT_CONFIDENCE
+
+
+def check_min_confidence(code, min_conf, tests):
+    v = core.Vulture(verbose=True, min_confidence=min_conf)
+    v.scan(code)
+    for item, (lineno, confidence) in zip(v.get_unused_code(), tests):
+        assert item.lineno == lineno
```

Comment:
For the unreachable code, the name is `return`, we won't be able to pass it as a keyword argument.